code/modules/genetics/genetics.dm

```diff
@@ -18,10 +18,10 @@
 *
 * ".Copy() just works"
 *
-* Example of an inherant_mutations list:
-* inherant_mutations= List(MUTATION_COW_SKIN, MUTATION_IMBECILE, MUTATION_MKNEWAIFUHAIR)
+* Example of an inherent_mutations list:
+* inherent_mutations= List(MUTATION_COW_SKIN, MUTATION_IMBECILE, MUTATION_MKNEWAIFUHAIR)
 **/
-//#define JANEDEBUG 1
+#define JANEDEBUG 1
```

Comment:
?

Reply:
oh. yeah. Might be a good idea to turn that off